vite.config.js

```diff
@@ -10,27 +10,41 @@ import pkg from './package.json';
  * see: https://github.com/hms-dbmi/viv/pull/469#issuecomment-877276110
  * @returns {import('vite').Plugin}
  */
-const bundleWebWorker = () => {
-    return {
-        name: 'bundle-web-worker',
-        apply: 'serve', // plugin only applied with dev-server
-        async transform(_, id) {
-            if (/\?worker_file$/.test(id)) {
-                // just use esbuild to bundle the worker dependencies
-                const bundle = await esbuild.build({
-                    entryPoints: [id],
-                    inject: ['./src/alias/buffer-shim.js'],
-                    format: 'esm',
-                    bundle: true,
-                    write: false,
-                });
-                if (bundle.outputFiles.length !== 1) {
-                    throw new Error('Worker must be a single module.');
-                }
-                return bundle.outputFiles[0].text;
+const bundleWebWorker = {
+    name: 'bundle-web-worker',
+    apply: 'serve', // plugin only applied with dev-server
+    async transform(_, id) {
+        if (/\?worker_file$/.test(id)) {
+            // just use esbuild to bundle the worker dependencies
+            const bundle = await esbuild.build({
+                entryPoints: [id],
+                inject: ['./src/alias/buffer-shim.js'],
+                format: 'esm',
+                bundle: true,
+                write: false,
+            });
+            if (bundle.outputFiles.length !== 1) {
+                throw new Error('Worker must be a single module.');
             }
-        },
-    };
+            return bundle.outputFiles[0].text;
+        }
+    },
+};
+
+/**
+ * Transforms /node_modules/.vite/* to include an alias for global.
+ * Only enabled during development.
+ *
+ * @type {import('vite').Plugin}
+ */
+const injectGlobal = {
+    name: 'inject-global',
+    apply: 'serve', // only runs during
+    transform(code, id) {
+        if (/node_modules/.test(id)) {
```

Comment:
Should this be something like the following?

```suggestion
        if (/node_modules\/\.vite\//.test(id)) {
```

It looks `global` is added to files outside `/node_modules/.vite/` (e.g., `/node_modules/monaco-editor/esm/vs/editor/standalone/browser/quickInput/standaloneQuickInput.css`), breaking styles of editor:

<img width="750" alt="Screen Shot 2021-09-24 at 5 20 49 PM" src="https://user-images.githubusercontent.com/9922882/134743175-e14fadce-8219-4428-9502-16606fbece52.png">

Reply:
I updated the regex to retrict to only `/node_modules/.vite/*.js` imports